beacon-chain/core/validators/committees.go

```diff
@@ -5,16 +5,24 @@
 package validators
 
 import (
+	"encoding/binary"
 	"fmt"
 
 	"github.com/prysmaticlabs/prysm/beacon-chain/utils"
 	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
 	"github.com/prysmaticlabs/prysm/shared/bitutil"
+	"github.com/prysmaticlabs/prysm/shared/bytes"
 	"github.com/prysmaticlabs/prysm/shared/mathutil"
-	"github.com/prysmaticlabs/prysm/shared/params"
 )
 
+// CrosslinkCommittee defines the validator committee of slot and shard combinations.
+type CrosslinkCommittee struct {
+	Committee []uint32
```

Comment:
Why is this slightly different than the ShardCommittee? https://github.com/prysmaticlabs/prysm/blob/master/proto/beacon/p2p/v1/types.proto#L140

Reply:
By slightly different, i mean it is missing a property `total_validator_count`

Reply:
`total_validator_count` was no longer needed/used. It was removed in the spec hence I removed it for `CrosslinkCommittee`

`pb.ShardCommitteeArray` and `pb.ShardCommittee` will be removed when #1352 is done

Reply:
Will a crosslink committee be represented in any proto? I'm wondering if we can use the proto definition or if we should use the struct here.

Reply:
crosslink committee will not be used in any proto, I thought of leaving it in proto and decided not to because of that